apps/native/app/src/stores/auth-store.ts

```diff
@@ -166,27 +188,25 @@ export const authStore = create<AuthStore>((set, get) => ({
 
 export const useAuthStore = createUse(authStore)
 
-export async function readAuthorizeResult(): Promise<AuthorizeResult | null> {
+export async function readAuthorizeResult(): Promise<void> {
   const { authorizeResult } = authStore.getState()
 
-  if (authorizeResult) {
-    return authorizeResult as AuthorizeResult
-  }
+  if (authorizeResult) return
```

Comment:
~Not a fan of skipping braces as it has historically caused hard to spot bugs. Though code standard/eslint/prettier should cover this.

Reply:
Yeah man that just like your opinion man :)
https://www.youtube.com/watch?v=XkADiJCYS2k